locales/bg_BG/LC_MESSAGES/duckduckgo.po

```diff
@@ -2512,10 +2511,9 @@ msgctxt "settings"
 msgid "Opens results in new windows/tabs"
 msgstr "Отваря резултатите в нов прозорец/подпрозорец"
 
-#, fuzzy
 msgctxt "feedback form"
 msgid "Optional"
-msgstr "по избор"
```

Comment:
Likewise with this. We've got `optional` (lowercase O) so these seem to be pretty much correct, apart from capitalization.

Worth putting together a quick script to fix capitalization for `Optional` so we get to keep the translations? I can imagine this would only work for Latin- and Cyrillic- script locales. I don't know enough about other locales to tell if it'll work.